packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1938,6 +1951,29 @@ export default class Meeting extends StatelessWebexPlugin {
     });
   }
 
+  /**
+   * Internal function to set up a basic listener which listens for
+   * the inEvent and tiggers the outEvent passing the payload.
+   * @returns {undefined}
+   * @param {string} inEvent
+   * @param {string} outEvent
+   */
+  setupDefaultSelfListener(inEvent, outEvent) {
```

Comment:
i feel this event would make it difficult to debug , any thoughts , its better to explicitly call the event listener

Reply:
Fair enough, I'll change it.